src/utils/logger.ts

```diff
@@ -1,24 +1,26 @@
-import chalk from "chalk";
+import chalk, { Chalk } from "chalk";
 import { format, createLogger, transports } from "winston";
 
 import { hasColor } from "./helpers.js";
 
 export const errorSymbol = "ⓘ"; // used in ../test-utils/matchers.ts to check for errors in console output
 
+export type LogLevelToColorMap = Record<string, Chalk | ((msg: string) => string)>;
+
+const logLevelToColorMap: LogLevelToColorMap = {
```

Comment:
Maybe also name it like "logLevelFormatter"

Reply:
Ok, corrected